testsuite/MDAnalysisTests/test_persistence.py

```diff
@@ -36,10 +36,8 @@
 import shutil
 import warnings
 
-"""These tests need to be reworked for the new 363 topoloy system
 
-"""
-class SkipAtomGroupPickle(object):
+class TestAtomGroupPickle(object):
```

Comment:
When a Universe has a `anchor_name` AtomGroups serialized from it can only be unpickled against Universes with the same `anchor_name`. This is a list independent of `is_anchor`, which just signals that the universe can be unpickled onto if the AtomGroup has no `anchor_name` information.

The tests that use `self.universe_n` check for correct universe selection when the AtomGroup comes from a Universe with `anchor_name` set. Also, in `test_unpickle_reanchor_other`, `self.universe_n` is made an anchor and it can then receive an unnamed AtomGroup.

Should we choose better flag names for the API?